### PR TITLE
riak-2.1.1: new package

### PR DESCRIPTION
**Pull request by cstrahan**
This packages Riak 2.1.1. ~~I still need to create a NixOS service to test this out, but it does build.~~

### CODE REVIEW

**Comment by jagajaga** [comment]
Any updates? :hamster:

**Comment by cstrahan** [comment]
Nope. I hope to get back to it soonish, though.

**Comment by cstrahan** [comment]
This seems to be working now. Or, at least, the console comes up, and I can use `riak ping` to connect and assert that the node is up and running.

I tried writing a test, but `$master->succeed('riak ping');` always fails, despite the node appearing to be up. Normally `riak ping` prints a message to `stderr` on failure, but there isn't any output in the test. Not sure what's going on there, but if someone else with an interest in Riak wants to take a stab at writing a test, I'd appreciate it.

File: nixos/modules/services/databases/riak.nix [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.riak = {
+
+      enable = mkOption {
```
Comment (jagajaga) [inline nixos/modules/services/databases/riak.nix]: Maybe change it to `mkEnableOption`?

**Comment by cstrahan** [comment]
Ok, I managed to fix the test. Redirecting stderr to stdout allowed me to see the problem: `succeed(...)`  doesn't  inherit the environment variables from the unit, so I had to supply those, otherwise `riak ping` doesn't know how to locate the server.

So, now we have working test too.

**Comment by jagajaga** [comment]
Awesome :+1:

**Pull request merged**